.github/workflows/docker_72_publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Docker publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker_build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Docker login
+        run: docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Hi @dmyers, thanks for your merging this PR.

And it should add `DOCKER_USER ` and `DOCKER_PASSWORD` repository secrets so that your Docker image will be published during GitHub Actions successfully.

More details about encrypted secret settings are available [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets).

Reply:
Here is my [successful building log](https://github.com/open-source-contributions/laravel-docker/runs/1647535271?check_suite_focus=true) on my forked repository.

And the targeted publishing Docker image URL is available [here](https://hub.docker.com/r/peter279k/laravel/tags?page=1&ordering=last_updated).